pkgs/by-name/ka/kargo/package.nix

```
@@ -0,0 +1,72 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  installShellFiles,
+  stdenv,
+}:
+
+buildGoModule rec {
+  pname = "kargo";
+  version = "1.3.0";
+
+  src = fetchFromGitHub {
+    owner = "akuity";
+    repo = "kargo";
+    tag = "v${version}";
+    hash = "sha256-GC3u4v8qrVi8+Ne08f074DvPh0RFmpmCOdJ5XD3kk+o=";
+  };
+
+  vendorHash = "sha256-Xb+9zu2uivOYETtz3ryMnBUJ3gJ/1ta1dLEpsD00jpU=";
+
+  subPackages = [ "cmd/cli" ];
+
+  ldflags =
+    let
+      package_url = "github.com/akuity/kargo/internal/version";
+    in
+    [
+      "-s"
+      "-w"
+      "-X ${package_url}.version=${src.rev}"
```

```suggestion
      "-X ${package_url}.version=${version}"
```
`src.rev` would be generated as `refs/tags/${src.tag}`